test/Functional/cypress-commands/src/test-runner.js

```diff
@@ -57,7 +57,7 @@ export function e2e(dir, assembly, { appDataLocation='./App_Data', dotnetVersion
   deleteDirectory(path.join(dir, "App_Data_Tests"));
   var server = host(dir, assembly, { appDataLocation, dotnetVersion });
 
-  let test = child_process.exec("npx cypress run");
+  let test = child_process.exec("npx cypress run --browser chrome");
```

Comment:
Can you make the --browser flag a parameter of the method defaulting to chrome ?

Reply:
By default it always uses Electron.

https://docs.cypress.io/guides/guides/launching-browsers.html#Electron-Browser